js/ui/appDisplay.js

```diff
@@ -2747,6 +2747,10 @@ class HackAppIcon extends AppIcon {
 
         let app = Clubhouse.getClubhouseApp();
         this._activated = false;
+        this._flatpak_link = Gio.File.new_for_path('/var/lib/flatpak/app/com.hack_computer.Clubhouse/current/active');
+        this._monitor = this._flatpak_link.monitor(0, null);
+        if (this._monitor)
+            this._monitor.connect('changed', this._checkNewRelease.bind(this));
```

Comment:
Does this need to be cleaned up somewhere? It looks to me like this creates a reference cycle.